src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
 /**
  * Parameterize this service and add its implementation in a separate class.
  */
-public interface MachineService {
+public interface MachineService<T extends Machine> { //useless T
```

Comment:
why is it useless? remove the comment

Reply:
IDE marks this T like unused (grey color)